### PR TITLE
FOGL-7159 Read the docs documentation added & other textual changes

### PR DESCRIPTION
**Pull request by ashish-jabble**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,22 @@
 # fledge-north-opcuaclient
 
+A Python plugin for Fledge that sends data north to with an OPC UA Client to a OPC UA Server.
```
Comment (RVerhoeff) [inline README.md]: Fix: to with an OPC UA Client

File: docs/index.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
+.. Images
+.. |opcuaclient_1| image:: images/opcuaclient.jpg
+
+
+North OpcuaClient
```
Comment (RVerhoeff) [inline docs/index.rst]: I think this should be _OPC UA Client North Plugin_

File: docs/index.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+  - Click on *Next*
+
+  - Configure the plugin
+
+  +-----------------+
+  | |opcuaclient_1| |
+  +-----------------+
+
+      - **OPCUA Server URL**: The url of the OPC UA Server from which data will be extracted. The URL should be of the form opc.tcp://...
```
Comment (RVerhoeff) [inline docs/index.rst]: ...OPC UA Server to which data will be sent.

File: docs/index.rst [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+      - **Register Map**: The register map defines which are set of locations in the OPC UA object hierarchy.
```
Comment (RVerhoeff) [inline docs/index.rst]: You should elaborate on the structure of the JSON. The outer names are Asset names. The inner names are Datapoint names.

**Review by RVerhoeff — CHANGES_REQUESTED** [review]
A few recommendations

File: docs/index.rst [hunk 5]
```diff
@@ -27,9 +27,25 @@ The plugin may be used within a north *task* or *service*. Both of these are cre
   | |opcuaclient_1| |
   +-----------------+
 
-      - **OPCUA Server URL**: The url of the OPC UA Server from which data will be extracted. The URL should be of the form opc.tcp://...
+      - **OPC UA Server URL**: The url of the OPC UA Server to which data will be sent. The URL should be of the form opc.tcp://...
 
-      - **Register Map**: The register map defines which are set of locations in the OPC UA object hierarchy.
+      - **Map**: A map for asset datapoints/attributes to OPC UA node objects. A map JSON structure be like the outer names are Asset names and the inner names are Datapoint names.
```
Comment (RVerhoeff) [inline docs/index.rst]: be like -> in which

**Review by RVerhoeff — APPROVED** [review]
One minor change, otherwise good.

File: docs/index.rst [hunk 6]
```diff
@@ -0,0 +1,55 @@
+.. Images
+.. |opcuaclient_1| image:: images/opcuaclient.jpg
+
+
+North OPC UA Client
+===================
+
+The *fledge-north-opcuaclient* plugin allows data to be sent with an OPC UA Client to a OPC UA Server.
```
Comment (MarkRiddoch) [inline docs/index.rst]: an OPC UA server

**Review by MarkRiddoch — CHANGES_REQUESTED** [review]

**Review by RVerhoeff — APPROVED** [review]

**Review by MarkRiddoch — APPROVED** [review]

**Pull request merged**